The merge process is blocked due to the error below, can you please fix this?
tensorflow/compiler/tf2tensorrt/convert/utils.cc:169:3: error: ignoring return value of function declared with 'warn_unused_result' attribute [-Werror,-Wunused-result]
  TensorShapeUtils::MakeShape(trt_dims.data(), trt_dims.size(), &shape);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
tensorflow/compiler/tf2tensorrt/convert/utils.cc:179:3: error: ignoring return value of function declared with 'warn_unused_result' attribute [-Werror,-Wunused-result]
  TensorShapeUtils::MakeShape(trt_dims.d, trt_dims.nbDims, &shape);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2 errors generated.

--- a/tensorflow/compiler/tf2tensorrt/convert/utils.cc
+++ b/tensorflow/compiler/tf2tensorrt/convert/utils.cc
@@ -163,24 +163,26 @@ bool AreShapesCompatible(const std::vector<TensorShape>& actual_shapes,
   return true;
 }
 
-TensorShape TrtDimsToTensorShape(const std::vector<int>& trt_dims,
-                                 bool use_implicit_batch, int batch_size) {
-  TensorShape shape;
-  TensorShapeUtils::MakeShape(trt_dims.data(), trt_dims.size(), &shape);
+Status TrtDimsToTensorShape(const std::vector<int>& trt_dims,
+                            bool use_implicit_batch, int batch_size,
+                            TensorShape& shape) {
+  TF_RETURN_IF_ERROR(
+      TensorShapeUtils::MakeShape(trt_dims.data(), trt_dims.size(), &shape));
   if (use_implicit_batch) {
     shape.InsertDim(0, batch_size);
   }
-  return shape;
+  return Status::OK();
 }
 
-TensorShape TrtDimsToTensorShape(const nvinfer1::Dims trt_dims,
-                                 bool use_implicit_batch, int batch_size) {
-  TensorShape shape;
-  TensorShapeUtils::MakeShape(trt_dims.d, trt_dims.nbDims, &shape);
+Status TrtDimsToTensorShape(const nvinfer1::Dims trt_dims,
+                            bool use_implicit_batch, int batch_size,
+                            TensorShape& shape) {
+  TF_RETURN_IF_ERROR(
+      TensorShapeUtils::MakeShape(trt_dims.d, trt_dims.nbDims, &shape));
   if (use_implicit_batch) {
     shape.InsertDim(0, batch_size);
   }
-  return shape;
+  return Status::OK();
 }
 
 int GetNumberOfEngineInputs(const nvinfer1::ICudaEngine* engine) {

--- a/tensorflow/compiler/tf2tensorrt/convert/utils.h
+++ b/tensorflow/compiler/tf2tensorrt/convert/utils.h
@@ -98,11 +98,13 @@ inline nvinfer1::Dims TensorShapeToTrtDims(const TensorShapeType& shape,
   return trt_dims;
 }
 
-TensorShape TrtDimsToTensorShape(const std::vector<int>& trt_dims,
-                                 bool use_implicit_batch, int batch_size = 1);
+Status TrtDimsToTensorShape(const std::vector<int>& trt_dims,
+                            bool use_implicit_batch, int batch_size,
+                            TensorShape& shape);
 
-TensorShape TrtDimsToTensorShape(const nvinfer1::Dims trt_dims,
-                                 bool use_implicit_batch, int batch_size = 1);
+Status TrtDimsToTensorShape(const nvinfer1::Dims trt_dims,
+                            bool use_implicit_batch, int batch_size,
+                            TensorShape& shape);
 
 // Returns a string that includes compile time TensorRT library version
 // information {Maj, Min, Patch}.

--- a/tensorflow/compiler/tf2tensorrt/utils/trt_engine_utils.cc
+++ b/tensorflow/compiler/tf2tensorrt/utils/trt_engine_utils.cc
@@ -51,7 +51,8 @@ Status GetTrtBindingShape(const nvinfer1::ICudaEngine* cuda_engine,
         "Explicit batch mode is only supported with TensorRT 6 and above.");
 #endif
   }
-  shape = TrtDimsToTensorShape(dims, use_implicit_batch, batch_size);
+  TF_RETURN_IF_ERROR(
+      TrtDimsToTensorShape(dims, use_implicit_batch, batch_size, shape));
   return Status::OK();
 }
 

